src/discordtipbot/discordbot.cr

```diff
@@ -122,10 +122,52 @@ class DiscordBot
     end
   end
 
+  private def split(msg : String, spliterator : String, max_length : Int32)
```

Comment:
I have no idea what a `spliterator` is. Could you come up with a more descriptive name for this argument? This should also accept `String | Char`.

`max_length` is pretty much always going to be `2000`. Unsure when you would use any other value. Consider just making this a constant, but at the very least make this a default value so it can be omitted from probably 99% of calls to this method.